docs/guidance/map/awards_contracts.md

```diff
@@ -145,36 +145,48 @@ There are no identifiers for the individual supplier/value pairings on the origi
 
 [View the example in Paraguay's API](https://contrataciones.gov.py/datos/api/v3/doc/ocds/record/ocds-03ad3f-340885-1)
 ```
+
 ## Purchase orders
 
 A purchase order is a specific type of contract, an official document issued by a buyer committing to pay a supplier for the supply of specific goods, services or works to be delivered in the future.
 
 Purchase orders can be issued against an existing contract, or if no prior contract exists then acceptance of a purchase order by a supplier forms a contract between buyer and supplier.
 
-Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to.
+Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to. Instead they ought to be disclosed in the `implementation.transactions` array of the `Contract`.
 
-### Example: Double counting contracts and purchase orders
+### Example: Purchase orders
 
-The UK's Department for Transport awards a £1.2m, 12-month contract to KPMG to provide the Project Management Office function for a project to construct a new highway bypass. The contract specifies that payment will be made quarterly in arrears in four equal amounts. The contract is represented in the `contracts` section of OCDS as follows:
+The Bureau of Parks and Recreation in Portland, Oregon, USA, awards a 2 year contract worth up to $1m to a local contractor to maintain Mt. Tabor Park. This contract specifies the goods, services and works to be provided along with unit prices and estimates of quantities. The contract is represented in the `contracts` section of OCDS as follows:
```

Comment:
Copy-edit to remove information that isn't relevant to what the example is intended to illustrate.

```suggestion
A buyer awards a contract for park maintenance services with a maximum value of $1 million to a single supplier. The contract specifies the services, unit prices, and estimated quantities. It is added to the `contracts` array.
```

Reply:
no longer relevant as the example has been completed changed